src/RPCClient.js

```diff
@@ -29,7 +31,11 @@ async function request(url, method, params, options = {}) {
   }
   const { data } = res;
   if (data.error) {
-    throw new Error(`DAPI RPC error: ${method}: ${data.error.message}`);
+    if (data.error.data) {
```

Comment:
We don't need this condition. In case if it's undefined you just pass the undefined into the RPCError constructor as an optional argument.

Reply:
changed